### PR TITLE
Css improvements

### PR DESCRIPTION
**Pull request by Juraci**
- [X] Grid set to Fluid for better spacing on small resolutions
- [X] App Bar no longer fixed due to broken behavior on small devices
- [X] Displaying select simulation only if it has enough space
- [X] Added a rotate device icon and tooltip for better user experience

screenshot_20161230-140350
screenshot_20161230-140448
screenshot_20161230-140526

### CODE REVIEW

**Comment by waldemarnt** [comment]
shipit

File: src/App/index.jsx [hunk 1]
```diff
@@ -31,9 +29,8 @@ export class App extends Component {
           <AppBar
             title="Sower"
             iconElementRight={icon}
-            style={barStyle}
           />
-          <Grid>
+          <Grid fluid>
```
Comment (lucaslago) [inline src/App/index.jsx]: is this synthax correct?

File: src/DashboardItem/index.test.jsx [hunk 2]
```diff
@@ -93,10 +93,6 @@ describe('<DashBoardItem />', () => {
       expect(toJson(wrapper)).toMatchSnapshot();
     });
 
-    it('renders the select-simulation component', () => {
-      expect(wrapper.find('.select-simulation').length).toBe(1);
```
Comment (lucaslago) [inline src/DashboardItem/index.test.jsx]: Why was this test case removed?

File: src/App/index.jsx (hunk 1 above)
Comment (Juraci) [inline src/App/index.jsx]: this is the shorthand syntax when you have a boolean prop set to true:
fluid={true} === fluid

File: src/DashboardItem/index.test.jsx (hunk 2 above)
Comment (Juraci) [inline src/DashboardItem/index.test.jsx]: The mediaquery prevents the component of being displayed with shallow and mount. But don't worry there is a test for this logic here:
https://github.com/agco-fuse/sower-ui/blob/master/src/SelectSimulation/index.test.jsx#L4-L37

File: src/App/index.jsx (hunk 1 above)
Comment (lucaslago) [inline src/App/index.jsx]: Ohh yeahh, I forgot about that 👍

File: src/DashboardItem/index.test.jsx (hunk 2 above)
Comment (lucaslago) [inline src/DashboardItem/index.test.jsx]: 👍

**Pull request merged**